rites/roman1969/src/locales/en.ts

```diff
@@ -386,6 +386,9 @@ export const locale: Locale = {
       'The Dedication of the Cathedral Saint Maurice of Angers, France', // mr_fr_2022_ed3_angers
     dedication_of_the_cathedral_of_saint_stephen_of_toulouse_france:
       'The Dedication of the Cathedral of Saint Stephen of Toulouse, France',
+    // src: http://catholic-dlc.org.hk/English_Liturgy.htm, http://catholic-dlc.org.hk/st/Dec/9/9.htm
+    dedication_of_the_cathedral_of_the_immaculate_conception_hong_kong:
```

Comment:
I would append `_china` (the country of which the diocese is part of) to the localisation key name.

```suggestion
    dedication_of_the_cathedral_of_the_immaculate_conception_hong_kong_china:
```

Reply:
not being political, but I'd rather follow the official source(http://catholic-dlc.org.hk/viewer.htm?doc=1209_Dedication_of_the_Cathedral-enA5.doc) which does not have China at the end.

Reply:
@ramishi, in this case, I am talking about the property _name_ (i.e. `dedication_of_the_cathedral_of_the_immaculate_conception_hong_kong_china`), not the property _value_ (e.g. `Dedication of the Cathedral of the Immaculate Conception, Hong Kong`); I agree the value must be the same as is used in the official sources, however, the property names have some (unofficial) rules which should be followed. And we also have some precedents, e.g. `dedication_of_the_cathedral_of_saint_stephen_of_toulouse_france`.

That said, I see a possible tension here, therefore, I think we must define which diocese is part of which country, ideally using official ecclesiastical documents.

Reply:
agreed